src/v/pandaproxy/client/test/consumer_group.cc

```diff
@@ -326,6 +327,78 @@ FIXTURE_TEST(pandaproxy_consumer_group, ppc_test_fixture) {
         }
     }
 
+    info("Producing data");
+    const int batch_size = 3;
+    using offsets_t = absl::flat_hash_map<
+      model::topic,
+      absl::flat_hash_map<model::partition_id, model::offset>>;
+    offsets_t produced_offsets;
+    for (const auto& t : topics) {
+        for (model::partition_id p{0}; p < partition_count; ++p) {
+            auto tp = model::topic_partition{t, p};
+            auto res = client
+                         .produce_record_batch(
+                           tp, make_batch(model::offset{0}, batch_size))
+                         .get();
+            BOOST_REQUIRE_EQUAL(res.error, kafka::error_code::none);
+            produced_offsets[t][p] = res.base_offset
+                                     + model::offset(batch_size - 1);
+            info("\033[1;31mProduced {}, {}\033[0m", tp, res.base_offset);
```

Comment:
haha. i don't think we use color logs anywhere. I'm ok w/ it. but seems hard to read IMO.

Reply:
It makes it easy to read for debugging, though! I intended to drop it. Next time I'll use https://fmt.dev/latest/api.html#color-api :)